### PR TITLE
[FEATURE] Literalinclude Directive

### PR DESCRIPTION
**Pull request by linawolf**
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude

### CODE REVIEW

File: lib/Directives/Literalinclude.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+        $path = $parser->getEnvironment()->absoluteRelativePath(trim($data));
+        if (! file_exists($path)) {
+            $configuration->getErrorManager()
+                ->warning(sprintf('Literalinclude "%s" does not exist or is not readable.', $data));
+
+            return;
+        }
+
+        $contents = (string) file_get_contents($path);
```
Comment (greg0ire) [inline lib/Directives/Literalinclude.php]: I'd use either `assert($contents !== false)` or even throw an exception.

File: tests/Functional/FunctionalTest.php [hunk 2]
```diff
@@ -201,7 +201,8 @@ private function findSubDirectories(string $directory): iterable
 
         return $finder
             ->directories()
-            ->in($directory);
+            ->in($directory)
+            ->notName('_*');
```
Comment (greg0ire) [inline tests/Functional/FunctionalTest.php:205]: Let's document this trick in `CONTRIBUTING.md` or wherever you see fit?

Comment (linawolf) [reply tests/Functional/FunctionalTest.php:205]: Until now we have not documented at all how to create Functional tests in our project. Not sure if it needs to be documented either as there are many things that also need to be documented and would help a broader number of developers. 
So if we document it I would do it in the PR that documents how to add tests in General

Comment (greg0ire) [reply tests/Functional/FunctionalTest.php:205]: Ok, I'll create an issue to track this.

**Review by greg0ire — APPROVED** [review]

File: lib/HTML/HTMLFormat.php [hunk 3]
```diff
@@ -44,6 +47,28 @@ public function getDirectives(): array
...
+            new WrapperTextRole('command', '<strong class="command">%s</strong>'),
+            new WrapperTextRole('dfn', '<em class="dfn">%s</em>'),
+            new WrapperTextRole('file', '<span class="pre file">%s</span>'),
+            new WrapperTextRole('guilabel', '<span class="guilabel">%s</span>'),
+            new WrapperTextRole('kbd', '<kbd class="kbd docutils literal notranslate">%s</kbd>'),
+            new WrapperTextRole('mailheader', '<em class="mailheader">%s</em>'),
+            new WrapperTextRole('math', '<math>%s</math>'),
+            new WrapperTextRole('emphasis', '<em>%s</em>'),
+            new WrapperTextRole('literal', '<span class="pre">%s</span>'),
+            new WrapperTextRole('strong', '<em class="strong">%s</em>'),
```
Comment (SenseException) [inline lib/HTML/HTMLFormat.php]:
```suggestion
            new WrapperTextRole('strong', '<strong>%s</strong>'),
```

**Review by SenseException — CHANGES_REQUESTED** [review]

File: lib/HTML/HTMLFormat.php (hunk 3 above)
Comment (linawolf) [reply lib/HTML/HTMLFormat.php]: Let me rebase this properly

**Review by SenseException — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @linawolf !

**Pull request merged**